provider/mediaconvert/audio_mapping.go

```diff
@@ -0,0 +1,83 @@
+package mediaconvert
+
+import (
+	"github.com/aws/aws-sdk-go-v2/service/mediaconvert"
+	"github.com/cbsinteractive/transcode-orchestrator/db"
+)
+
+const (
+	muteChannel   int64  = 0
+	setChannel    int64  = -60
+	lfeChannel    string = "LFE"
+	centerChannel string = "C"
+	leftChannel   string = "L"
+	rightChannel  string = "R"
+)
+
+func (p *mcProvider) audioSelectorsFrom(audioDownmix db.AudioDownmix) map[string]mediaconvert.AudioSelector {
+	audioSelector := mediaconvert.AudioSelector{
+		DefaultSelection: mediaconvert.AudioDefaultSelectionDefault,
+	}
+
+	if audioDownmix.IsSet() {
+		var offset int64
+		var programSelection int64 = 1
+		var channelsIn int64 = int64(len(audioDownmix.SrcChannels))
+		var channelsOut int64 = int64(len(audioDownmix.DestChannels))
+
+		audioSelector.Offset = &offset
+		audioSelector.SelectorType = mediaconvert.AudioSelectorTypeTrack
+		audioSelector.ProgramSelection = &programSelection
+
+		for i := 0; i < len(audioDownmix.SrcChannels); i++ {
+			audioSelector.Tracks = append(audioSelector.Tracks, int64(i+1))
+		}
+
+		audioSelector.RemixSettings = &mediaconvert.RemixSettings{
+			ChannelsIn:     &channelsIn,
+			ChannelsOut:    &channelsOut,
```

Comment:
tiny nit, we could avoid creating variables for the channel in/out by doing something like
```suggestion
			ChannelsIn:	aws.Int64(int64(len(audioDownmix.SrcChannels))),
			ChannelsOut:	aws.Int64(int64(len(audioDownmix. DestChannels))),